classes/task/ReviewReminder.inc.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/**
+ * @file classes/task/ReviewReminder.inc.php
+ *
+ * Copyright (c) 2013-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ReviewReminder
+ * @ingroup tasks
+ *
+ * @brief Class to perform automated reminders for reviewers.
+ */
+
+import('lib.pkp.classes.scheduledTask.ScheduledTask');
+
+define('REVIEW_REMIND_AUTO', 'REVIEW_REMIND_AUTO');
+define('REVIEW_REQUEST_REMIND_AUTO', 'REVIEW_REQUEST_REMIND_AUTO');
```

Comment:
I am not sure if I understand it correctly -- what do you mean by "invitation reminders"? -- but I believe yes :-)
Now there is REVIEW_REQUEST, REVIEW_REQUEST_REMIND_AUTO... and REVIEW_REMIND and REVIEW_REMIND_AUTO...